EntAppProject/JBCreateAccount.html

```diff
@@ -1,53 +1,33 @@
-<!DOCTYPE html>
-
-<html>
-
-<head>
-
-<title>JobBook - Create an Account</title>
-
-<link rel="stylesheet" href="JobBookCSS.css">
- 
-</head>
-
-<body>
- 
-<h1><span>Job</span>Book</h1>
-
-<h2>Create an Account</h2>
-
-<br>
-
-<fieldset>
-
-<form action="file:///C:/Users/Administrator/eclipse-workspace/jobBookFINAL/src/main/resources/templates/JBHomescreen.html" method="post">
-
-<div class="container">
-        <label for="useremail"><b>Email:</b></label>
-        <input type="email" placeholder="Email:" name="useremail" required>
-        <br>
-        <label for="username"><b>Username:</b></label>
-        <input type="text" placeholder="Username:" name="username" required>
-        <br>
-        <br>
-        <label for="userpassword"><b>Password:</b></label>
-        <input type="password" placeholder="Password" name="userpassword" required>
-        <br>
-        <label for="userpasswordconfirm"><b>Confirm Password:</b></label>
-        <input type="password" placeholder="Password" name="userpasswordconfirm" required>
-        <br>
-        <button type="submit">Create</button>
-        <label>
-        <br>
-        <input type="checkbox" checked="checked" name="remember"> Remember me?
-        </label>
-        </div>
-        <br>
-        <br>
-        </form>
- 
-</fieldset>
- 
-</body>
-
+<!DOCTYPE html>
+
+<html>
+
+        <head>
+                <title>JobBook - Create an Account</title>
+                <link rel="stylesheet" href="JobBookCSS.css">
+        </head>
+
+        <body>
+                <h1><span>Job</span>Book</h1>
+                <h2>Create an Account</h2>
+                <br>
+
+                <form action="file:///C:/Users/Administrator/eclipse-workspace/jobBookFINAL/src/main/resources/templates/JBHomescreen.html" method="post">
+                        <fieldset>
```

Comment:
the `<fieldset>` tag is usually placed within a form on an HTML page. It is usually meant to specify that it contains fields of a form/input boxes. I usually use this for stylistic purposes regarding the UI, but I believe it has other uses as well. Regardless, I moved `<fieldset>` inside of the form tag.